logger/unilog.go

```diff
@@ -112,7 +115,8 @@ func (u *Unilog) addFlags() {
 	flag.StringVar(&u.SentryDSN, "sentrydsn", u.SentryDSN, "Sentry DSN to send errors to")
 	flag.StringVar(&u.StatsdAddress, "statsdaddress", "127.0.0.1:8200", "Address to send statsd metrics to")
 	flag.StringVar(&clevels.AusterityFile, "austerityfile", clevels.AusterityFile, "(optional) Location of file to read austerity level from")
-	stringFlag(&statstags, "statstags", "s", "", `(optional) tags to include with all statsd metrics (e.g. "foo:bar,baz:quz")`)
+	stringFlag(&statstags, "statstags", "s", "", `(optional) tags to include with all statsd metrics except those about the box's austerity levels (e.g. "foo:bar,baz:quz")`)
```

Comment:
now that we've added the exception, the "e.g." reads a bit weird. I'd say something like:

```
(optional) tags to include with all statsd metrics except those about the box's austerity levels (format: "foo:bar,baz:quz"). See also -cleveltags
```

and also maybe specify in the docstring for `cleveltags` which metric is affected (since I think it's only one or two).